db/migrate/20200124114452_add_primary_key_to_provider_users_providers.rb

```diff
@@ -0,0 +1,9 @@
+class AddPrimaryKeyToProviderUsersProviders < ActiveRecord::Migration[6.0]
+  def change
+    change_table :provider_users_providers, bulk: false do |t|
+      t.primary_key :id
+      t.datetime :created_at, null: false, default: -> { 'current_timestamp' }
+      t.datetime :updated_at, null: false, default: -> { 'current_timestamp' }
```

Comment:
Would `add_timestamps` work here, or do we need the default to backfill?

https://apidock.com/rails/v2.1.0/ActiveRecord/ConnectionAdapters/SchemaStatements/add_timestamps

Reply:
That doesn't seem to work - like you say it doesn't backfill.